redisinsight/values.yaml

```diff
@@ -0,0 +1,125 @@
+# Default values for redisinsight.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- The name of the environment
+platform: sandbox
```

Comment:
**suggestion:** I think we should remove the `platform` value. This should not be related with `redisInsight`.

In general, we would not want to define a platform/infrastructure parameter at a package level. The packages should remain agnostic to the environment it will be deployed to.

**thought:** We should probably also remove it from the `gpt-survey-summarizer`, but this is not part of this PR.